### PR TITLE
Landing page + signup page + errors solved

### PR DESCRIPTION
**Pull request by BhuvanSharma0711**
working fine on my device

### CODE REVIEW

File: apps/server/backend/src/main.ts [hunk 1]
```diff
@@ -6,10 +6,10 @@ import * as cookieParser from 'cookie-parser';
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
 
-  // app.enableCors({
-  //   origin: ['http://localhost:3000'],
-  //   credentials: true,
-  // });
+  app.enableCors({
+    origin: ['http://localhost:3001'],
```
Comment (Himasnhu-AT) [inline apps/server/backend/src/main.ts]: locally nextjs runs on 3000, update that

File: apps/web/src/components/Herosection.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        className="text-2xl px-4 md:text-4xl lg:text-5xl font-bold text-neutral-700 dark:text-white max-w-4xl leading-relaxed lg:leading-snug text-center mx-auto "
+      >
+           Experience the Future of Note-Taking.  Where AI Meets Your Ideas
+        {" "}
+        <Highlight className="text-black dark:text-white">
+         for Ultimate Clarity
+        </Highlight>
+      </motion.h1>
+      <TextGenerateEffect words={words} className="text-1xl"/> 
+      <div className="flex justify-center mt-10">
```
Comment (Himasnhu-AT) [inline apps/web/src/components/Herosection.tsx]: Fields not visible, give z index if hidden by background

File: apps/web/src/components/Navigationbar.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                </NavbarContent>
+
+                <NavbarContent className="hidden sm:flex gap-4" justify="center">
+                    <NavbarItem>
+                    <Link color="foreground" href="/">
+                        Homepage
+                    </Link>
+                    </NavbarItem>
+
+                    <NavbarItem isActive>
```
Comment (Himasnhu-AT) [inline apps/web/src/components/Navigationbar.tsx]: keep this filled as a state variable, or remove altogether

isActivate var one...

File: apps/web/src/components/Signup.tsx [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+      </p>
+
+      <form className="my-8" onSubmit={handleSubmit}>
+        <div className="flex flex-col md:flex-row space-y-2 md:space-y-0 md:space-x-2 mb-4">
+          <LabelInputContainer>
+            <Label htmlFor="firstname">First name</Label>
+            <Input
+              id="firstname"
+              name="firstName"
+              placeholder="Bhuvan"
```
Comment (Himasnhu-AT) [inline apps/web/src/components/Signup.tsx]: for placeholder, use fields related to noterai

**Review by Himasnhu-AT — CHANGES_REQUESTED** [review]
Resolve these comments, along with it, run `pnpm build` locally and ensure it builds

**Comment by BhuvanSharma0711** [comment]
it builds locally now without any errors

**Review by Himasnhu-AT — APPROVED** [review]

**Pull request merged**